packages/redux/src/selectors.ts

```diff
@@ -41,38 +41,71 @@ export const getWorkspaceStorage = <State extends RootState>(
   store: State
 ): State['storage']['workspaceStorage'] => getStorage(store).workspaceStorage;
 
+export const getDefaultWorkspaceSettings = <State extends RootState>(
+  store: State
+): State['defaultWorkspaceSettings'] => store.defaultWorkspaceSettings;
+
 // Workspace
 export const getWorkspace = <State extends RootState>(
   store: State
-): State['workspace'] => store.workspace;
+): Workspace => {
+  const { workspace } = store;
+
+  if (workspace == null) {
+    // this will only be null on initial render
+    return null as never;
+  }
+  const defaultInjectedWorkspace: Workspace = {
+    data: {
+      ...workspace.data,
+      settings: getDefaultWorkspaceSettings(store) ?? {},
+    },
+  };
```

Comment:
I wonder if we are fetching the entire workspace then manually destructure the settings object from it somewhere. My original intention was so that the Workspace object would look identical to before the change, to avoid creating unintended changes elsewhere